index.html

```diff
@@ -23,29 +23,227 @@
                 </a>
             </div>
         </nav>
-    </header>
-    <main class="main">
-        <section class="headline">
-            <h2 class="headline-text">I’m Belay <br> Glad to see <br> you! </h2>
-            <p class="description-text"> I can help you build a product , feature or website
+        <div class="headline">
+          <div class="headline-container">
+            <div class="card-header">
+              <h2 class="headline-text">I’m Belay <br> Glad to see <br> you! </h2>
+              <p class="description-text"> I can help you build a product , feature or website
                 Look through some of my work and experience! If you
                 like what you see and have a project you need coded,
                 don’t hestiate to contact me.
-            </p>
-            <div>
-                <p class="social-icons-header">Let's Connect</p>
-                <ul class="social-icons">
+              </p>
+            </div>
+              <div class="card-body">
+                <div class="social-liks">
+                  <p class="social-icons-header">Let's Connect</p>
+                  <ul class="social-icons">
                     <li><a href="https://twitter.com/2belamit" aria-label="Twitter" target="_blank" rel="noopener"><i
-                                class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin"
-                            target="_blank" rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                          class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin" target="_blank"
+                        rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
                     <li><a href="https://github.com/belaymit?tab=repositories" aria-label="Github" target="_blank" rel="noopener"><i
-                                class="fa fa-github"></i></a></li>
-                    <li><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i
-                                class="fa-brands fa-angellist"></i></a></li>
-                </ul>
+                          class="fa fa-github"></i></a></li>
+                    <li><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i class="fa-brands fa-angellist"></i></a>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+          </div>
+        </div>
+    </header>
+  <main class="main">      
+    <section class="work-section">
+      <div class="card-header">
+        <div class="course-header">
+          <img src="./Images/Snapshoot Portfolio.svg" alt="" class="image">
+        </div>
+      </div>
+      <div class="card-body">
+        <div class="title">Tonic</div>

```

Comment:
This here should use a `<hx> from <h3> to <h6>` tag. This is because it is a heading and using the correct semantic makes your site usable for all as well as friendly to screen readers